packages/apollo-cli/src/commands/codegen/generate.ts

```diff
@@ -95,6 +95,10 @@ export default class Generate extends Command {
       description:
         'By default, TypeScript/Flow will put each generated file in a directory next to its source file using the value of the "output" as the directory name. Set "outputFlat" to put all generated files in the directory relative to the current working directory defined by "output".'
     }),
+    outputGlobalTypes: flags.string({
```

Comment:
Not doing a full review, but I think that `outputGlobalTypes` is a strange name (and reads like a boolean) for a string that is meant to specify a file location to write global type information to. Perhaps something like `globalTypesFilePath`?

Reply:
The intention was that it is aligned with the other output options (`output` and `outputFlat`) for easier grouping if they all begin with `output...`. But I really don't care what the param is named :-)

Reply:
I agree `outputGlobalTypes` feels a bit weird as a flag name. Maybe `globalTypesFile`?

Reply:
@martijnwalraven I amended the commit